Makefile

```diff
@@ -231,7 +231,7 @@ lint: .make/lint
 
 .PHONY: stylus-benchmarks
 stylus-benchmarks: $(stylus_benchmarks)
-	cargo test --manifest-path $< --release --features benchmark benchmark_ -- --nocapture
+	cargo test --manifest-path $< --release --features  benchmark_ -- --nocapture
```

Comment:
I think we should revert this change because that's what nitro has

https://github.com/OffchainLabs/nitro/blob/bcfdfcfdd9e74e314d2c37b95636d9fd39f68875/Makefile#L225